pywren/wren.py

```diff
@@ -60,7 +59,7 @@ def remote_executor(config=None, job_max_runtime=3600):
 
     AWS_REGION = config['account']['aws_region']
     SQS_QUEUE = config['standalone']['sqs_queue_name']
-    invoker = invokers.SQSInvoker(AWS_REGION, SQS_QUEUE)
+    invoker = queues.SQSInvoker(AWS_REGION, SQS_QUEUE)
 
```

Comment:
So to do this I think we need to import SQSInvoker into invokers.py ? If I do that then I get `W:  9, 0: Unused SQSInvoker imported from pywren.queues (unused-import)` in invokers.py.

If I dont do that import then I get `E: 61,14: Module 'pywren.invokers' has no 'SQSInvoker' member (no-member)`

Again -- I'm not sure having the class in queues.py is a good idea if we want to call it from invokers.py (if we need a module we need to create invokers as a directory ? that seems overkill)